lightly/transforms/rotation.py

```diff
@@ -50,10 +50,30 @@ def __call__(self, sample):
         return sample
 
 class RandomRotateDegrees(object):
+    """Random rotate image between two rotation angles with a random probability.
+
+    Attributes:
+        prob:
+            Probability with which image is rotated.
+        degrees (sequence or number): Range of degrees to select from.
+            If degrees is a number instead of sequence like (min, max), the range of degrees
+            will be (-degrees, +degrees).
```

Comment:
We use a different docstring format that pytorch. We don't put the type in the docstring as it is already documented in the type hints and we always put the description on a new line. Like this:
```
        degrees: 
            Range of degrees to select from. If degrees is a number instead of sequence
            like (min, max), the range of degrees will be (-degrees, +degrees).
```